content/guides/getting-started/testing-your-app.md

```diff
@@ -145,18 +145,27 @@ leverage that now.
 Open up your [configuration file](/guides/references/configuration)
 (`cypress.json` in your project directory, by default) It starts out empty:
 
-```json
-{}
+:::cypress-config-example
```

Comment:
as-is, an empty object will just show as below which might be ok for now

`const { defineConfig } = require('cypress')

module.exports = defineConfig({
})`

Reply:
I think, in this case, we don't need to show the "empty" example, but can instead say "It starts out empty, but let's add the `baseUrl` option." and then just show the example with `baseUrl` added.